protected/humhub/modules/admin/views/pending-registrations/index.php

```diff
@@ -7,42 +7,65 @@
 ?>
 
 <div class="panel-body">
-    <?= humhub\libs\Html::backButton(['/admin/user/index'], ['label' => Yii::t('AdminModule.base', 'Back to user overview'), 'class' => 'pull-right']); ?>
+    <?= humhub\libs\Html::backButton(
+        ['/admin/user/index'],
+        ['label' => Yii::t('AdminModule.base', 'Back to user overview'), 'class' => 'pull-right']
+    ); ?>
     <h4><?= Yii::t('AdminModule.base', 'Pending user registrations'); ?></h4>
 
     <div class="help-block">
-        <?= Yii::t('AdminModule.views_approval_index', 'The following list contains all pending sign-ups and invites.'); ?>
+        <?= Yii::t(
+            'AdminModule.views_approval_index',
```

Comment:
This shouldn't be on another line.

Reply:
According to codeclimate it should. 123 chars.